src/Simplic.OxS.Server/Filter/ValidationActionFilter.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Simplic.OxS.Server.Filter;
```

Comment:
while file scoped namespaced are a nice feature we like to have a simmilar style in all our files and it would currently be a lot of work to change all files.

So please do not use them at the current point.